### PR TITLE
W-9696590 Mulesoft GA: Update labels and documentation/demos to reference "Source API name"

### PR DESCRIPTION
**Pull request by matthew-zizheng-li**
https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07AH000000Sy8tYAC/view

### CODE REVIEW

File: docs-connectors.iml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (chrisflanigan) [inline docs-connectors.iml]: What is this IML file for?

Comment (chrisflanigan) [reply docs-connectors.iml:1]: Seems like an ADD - vs and EDIT?

File: salesforce-cdp/1.0/modules/ROOT/pages/salesforce-cdp-connector-examples.adoc [hunk 2]
```diff
@@ -30,13 +30,13 @@ This Mule flow shows how to upsert data via the streaming API to an object's end
 Here is what each operation is doing:
 
 * *HTTP Listener* +
-Accepts data from HTTP PUT requests with the connector and object name as the URI parameters.
+Accepts data from HTTP PUT requests with the source and object name as the URI parameters.
```
Comment (chrisflanigan) [inline salesforce-cdp/1.0/modules/ROOT/pages/salesforce-cdp-connector-examples.adoc]: @matthew-zizheng-li Should this be `HTTP POST`?

File: salesforce-cdp/1.0/modules/ROOT/pages/salesforce-cdp-connector-examples.adoc [hunk 3]
```diff
@@ -66,9 +66,9 @@ http://www.mulesoft.org/schema/mule/sdc http://www.mulesoft.org/schema/mule/sdc/
 	</sdc:sdc-config>
 
 	<flow name="upsert-objectsFlow" >
-		<http:listener doc:name="PUT /upsert/{connectorName}/{objectName}" config-ref="HTTP_Listener_config" path="/upsert/{connectorName}/{objectName}" allowedMethods="PUT"/>
+		<http:listener doc:name="PUT /upsert/{sourceApiName}/{objectName}" config-ref="HTTP_Listener_config" path="/upsert/{sourceApiName}/{objectName}" allowedMethods="PUT"/>
```
Comment (chrisflanigan) [inline salesforce-cdp/1.0/modules/ROOT/pages/salesforce-cdp-connector-examples.adoc:69]: Should this be `allowedMethods="POST"`?

File: salesforce-cdp/1.0/modules/ROOT/pages/salesforce-cdp-connector-examples.adoc (hunk 2 above)
Comment (chrisflanigan) [reply salesforce-cdp/1.0/modules/ROOT/pages/salesforce-cdp-connector-examples.adoc]: The HTTP trigger is doing PUT => POST?

Do we want to keep the protocol POST all the way through?

Comment (chrisflanigan) [reply salesforce-cdp/1.0/modules/ROOT/pages/salesforce-cdp-connector-examples.adoc]: I guess this is a bigger philosophy question...

File: docs-connectors.iml (hunk 1 above)
Comment (matthew-zizheng-li) [reply docs-connectors.iml]: Uploaded by accident.

**Review by chrisflanigan — APPROVED** [review]
@matthew-zizheng-li LGTM!

**Review by sathya0 — APPROVED** [review]

**Pull request merged**